tests/sv/cover_HassCloseCover.yaml

```diff
@@ -0,0 +1,37 @@
+language: sv
+tests:
+  - sentences:
+      - "stäng cloth"
```

Comment:
This is not a Swedish word. Cloth translated to Swedish is "trasa". You probably mean "Gardin" or "Gardinen" here (or something similar).

Reply:
This is probably because of me having an entity in English called Cloth. I have since removed that and replaced it with "Curtain Left". I would recommend naming your entity something similar in Swedish.